concepts/list-methods/about.md

```diff
@@ -112,7 +112,7 @@ You can reverse the order of a list with the `list.reverse()` method.
 [3, 2, 1]
 ```
 
-A list can be re-ordered _**in place**_ with the help of `list.sort()`. Internally, Python uses [`Timsort`][timsort] to arrange the list. The default order is _ascending_. Take a look at the Python docs for some [additional tips and techniques for sorting][sorting how to] lists effectively.
+List elements can also be sorted  _**in place**_ with the help of `list.sort()`. Internally, Python uses [`Timsort`][timsort] to arrange elements. The default order is _ascending_. Take a look at the Python docs for some [additional tips and techniques for sorting][sorting how to] lists effectively.
```

Comment:
```suggestion
List elements can also be sorted  _**in place**_ using `list.sort()`. Internally, Python uses [`Timsort`][timsort] to arrange elements. The default order is _ascending_. Take a look at the Python docs for some [additional tips and techniques for sorting][sorting how to] lists effectively.
```

Reply:
Are you using `<list>.sort()` or `list.sort()`? It is the former on L106.

Reply:
`<list>.sort()` to imply that `<list>` be replaced with the name/list in question.  But if the usage is unclear then we should change it.

Reply:
Committing this so I can then re-edit it.